src/resources/postcss/base/forms/_tabs.pcss

```diff
@@ -0,0 +1,117 @@
+.tribe-events {
+	/* -----------------------------------------------------------------------------
+	 *
+	 * Form Control: Tabs
+	 *
+	 * Example:
+	 * <div class="tribe-common-form-control-tabs">
+	 *   <button aria-haspopup="listbox" aria-labelledby="tabButton" id="tabButton">Tab One</button>
+	 *   <ul tabindex="-1" role="listbox">
+	 *     <li role="presentation">
+	 *       <input id="tabOne" name="tabGroup" type="radio" value="tabOne" checked="checked" />
+	 *       <label id="tabOneLabel" for="tabOne" role="option">Tab One</label>
+	 *     </li>
+	 *     <li role="presentation">
+	 *       <input id="tabTwo" name="tabGroup" type="radio" value="tabTwo" />
+	 *       <label id="tabTwoLabel" for="tabTwo" role="option">Tab Two</label>
+	 *     </li>
+	 *   </ul>
+	 * </div>
+	 *
+	 * ----------------------------------------------------------------------------- */
+
+	 /**
+	  * @todo: implement listbox JS
+		* https://www.w3.org/TR/wai-aria-practices/examples/listbox/listbox-collapsible.html
```

Comment:
Assuming this isn't for powering tabs? Do you have an a11y tab solution in mind?

Reply:
The tabs are radio buttons with radios hidden, and labels as the tabs. When the tabs go to mobile, they will turn into a dropdown.

One thought I had here was to have 2 different markups, one for the tabs (using radios) on desktop, one for dropdown using the list as you saw in the dropdown example, and hiding the other in each media query case. If the radios + labels can also serve as dropdown list items, then we won't need the second markup, but if it doesn't work as well (especially from a11y perspective) then we may need to use a second markup (or another approach). Thoughts @ryanurban?

Reply:
Needing 2 different sets of markup to hide or show based on mobile or desktop feels...icky : ) So the UI changes based on mobile or desktop for this UI?

Reply:
Definitely icky. Gotta think a bit about how best to implement this combined tab/dropdown component.